### PR TITLE
feat: receive collectible flow

### LINKED ISSUE #3091: Display fresh taproot address for depositing inscriptions with proper guidance
For inscribing or depositing Ordinals. E.g. under "Collectibles" as an "Add inscription" option with the following type of content displayed upon selection:

Screenshot 2023-02-17 at 02 38 40

[See related Figma](https://www.figma.com/file/OSQocEEJnfcMkniEl8u3U8/%231483---Initial-Bitcoin-support?node-id=1182%3A63497&t=Shl5eSB4xRFAbQkH-4)

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/4215941748).

This PR adds the flow to copy a taproot address to receive an ordinal.

Screen Shot 2023-02-18 at 1 43 38 PM

Screen Shot 2023-02-18 at 1 42 55 PM

### CODE REVIEW

**Comment by fbwoolf** [comment]
Note: This is NOT wired up to the taproot address yet. It will copy a segwit address.

**Comment by markmhendrickson** [comment]
Looks good! I'd just suggest these copy modifications and increasing the size of the Ordinals logo? cc @mica000 wdyt?

Screenshot 2023-02-18 at 12 36 34

File: src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { BaseDrawer } from '@app/components/drawer/base-drawer';
+import { ErrorLabel } from '@app/components/error-label';
+import { OrdinalIcon } from '@app/components/icons/ordinal-icon';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Text, Title } from '@app/components/typography';
+import { useCurrentBtcNativeSegwitAccountAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+export function ReceiveCollectibleOrdinal() {
```
Comment (kyranjamie) [inline src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx:16]: Would be great to make a reusable layout component afterwards

**Review by kyranjamie — APPROVED** [review]

**Comment by edu-stx** [comment]
Closing in favor of https://github.com/hirosystems/stacks-wallet-web/pull/3198. Work from this PR has been added there.

**Pull request closed without merging**